gameFunctions.js

```diff
@@ -0,0 +1,288 @@
+require('dotenv').config();
+
+const resFunc = require('./responseFunctions');
+const db = require('./dbUtils');
+const pollFuncs = require('./pollFunctions');
+const request = require('request');
+
+const startNewGame = (res, requestBody) => {
+    const newPoll = {
+        isGame: true,
+        isClosed: false,
+        choices: {
+            options: [{
+                    index: 1,
+                    name: 'Im in!',
+                    votes: []
+                },
+                {
+                    index: 2,
+                    name: 'Not this time.',
+                    votes: []
+                }
+            ]
+        },
+        teamId: requestBody.team_id,
+        channelId: requestBody.channel_id,
+        pollTitle: "New Game Anyone?"
+    };
+    db.createPoll(newPoll).then((result) => {
+        if (result.rowCount != 1) {
+            resFunc.sendErrorResponse(res);
+        } else {
+            resFunc.sendPublicResponse(res, `Poll Created!\n${pollFuncs.getFormattedPollResults(newPoll, false)}`);
+        }
+    }).catch(err => {
+        console.log(err);
+        resFunc.sendErrorResponse(res);
+    });
+}
+
+const closeNewGamePoll = (res, responseUrl, poll) => {
+    res.status(200).send();
+    const roles = assignRoles(poll, ['seer', 'doc', 'mason', 'mason']);
```

Comment:
I imagine we would want these roles to pull from the database in the long term?

Reply:
Correct, there's a trello card in for allowing the creator to choose roles and i was going to switch it there. This is first pass MVP of role assignment

Reply:
If @cjensen5011  gets his formatting working we could allow them to select roles from a list